### PR TITLE
Purescript 0.9.1

### PR DESCRIPTION
**Pull request by kika**
WIP, please do not merge yet. The bulk of work has been done, and code compiles, but tests were not run due to the lagging dependency `purescript-test-unit`
`purescript-node-fs` references my own fork, because the upstream hasn't updated the version tag yet.

### CODE REVIEW

File: examples/bower.json [hunk 1]
```diff
@@ -3,10 +3,10 @@
...
-    "purescript-console": "^0.1.1",
-    "purescript-node-fs": "^0.11.0",
-    "purescript-strings": "~0.7.1",
-    "purescript-refs": "~0.2.0",
-    "purescript-express": "master"
+    "purescript-console": "^1.0.0",
+    "purescript-node-fs": "^1.0.0",
+    "purescript-strings": "~1.0.0",
+    "purescript-refs": "~1.0.0",
+    "purescript-express": "https://github.com/kika/purescript-express.git#purescript-0.9.1"
```
Comment (dancingrobot84) [inline examples/bower.json]: Could you please change this reference back to `master`?

**Comment by dancingrobot84** [comment]
Thanks for the hard work, though it's a bit sad that tests are not updated. In this case I'll postpone releasing a new version, but instead just merge it to master so that you can use it.

**Comment by kika** [comment]
Done. Tests would take time, though, I have some commitments to fulfill and don't have much spare time now. 
@Risto-Stevcev, can you help?

**Comment by Risto-Stevcev** [comment]
@kika Sure, I can't dedicate a bunch of time on it because of other priorities, but I can refactor the tests.

**Pull request merged**